console/src/main/java/org/nervos/ckb/Wallet.java

```diff
@@ -26,21 +28,29 @@
 
   private static final String MIN_CAPACITY = "6000000000";
 
-  private Script lockScript;
   private String privateKey;
+  private Script lockScript;
+  private SystemScriptCell systemScriptCell;
   private CKBService ckbService;
 
-  public Wallet(String privateKey, Script lockScript, String nodeUrl) {
+  public Wallet(String privateKey, String nodeUrl) {
     this.privateKey = privateKey;
-    this.lockScript = lockScript;
+    HttpService.setDebug(false);
```

Comment:
what is the effect of this method?

Reply:
lockScript can generate from private and system script, so I remove it.

Add `HttpService.setDebug(false)` to make http log more clear.